### PR TITLE
French translations

### PR DESCRIPTION
**Pull request by bjoern-m**
# Description

- Adds French translations
- Adds back lost translations (for yes and no)

*Example:*

Script:
```
import { register } from "@teamhanko/hanko-elements";
import { fr } from "@teamhanko/hanko-elements/i18n/fr";

register("https://hanko.yourdomain.com", { translations: { fr } });
```

Markup:
```
<hanko-auth lang="fr"></hanko-auth>
```

Fixes https://github.com/teamhanko/hanko/issues/615

### CODE REVIEW

File: frontend/elements/src/i18n/fr.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import { Translation } from "./translations";
+
+export const fr: Translation = {
+  headlines: {
+    error: "Une erreur s'est produite",
+    loginEmail: "Se connecter ou s'inscrire",
+    loginFinished: "Connexion réussie",
+    loginPasscode: "Entrez le code d'accès",
+    loginPassword: "Entrez le mot de passe",
+    registerAuthenticator: "Enregistrer une passkey",
```
Comment (nicobao) [inline frontend/elements/src/i18n/fr.ts:10]: "Enregistrer une clé d'accès"

We could use the french term for passkey "Clé d'accès". Though the english term "passkey" IS used in French language, many French article mention it. And it is shorter.

File: frontend/elements/src/i18n/fr.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    registerConfirm: "Créer un compte ?",
+    registerPassword: "Définir un nouveau mot de passe",
+    profileEmails: "Adresses e-mail",
+    profilePassword: "Mot de passe",
+    profilePasskeys: "Passkeys",
```
Comment (nicobao) [inline frontend/elements/src/i18n/fr.ts:15]: Clés d'accès ?

**Review by nicobao — COMMENTED** [review]
Looks good to me apart from the "passkey" thing. After a second thought, I think it is fine to keep it this way. "Passkey" probably gives a more "modern" feel than "clé d'accès".

**Comment by bjoern-m** [comment]
> Looks good to me apart from the "passkey" thing. After a second thought, I think it is fine to keep it this way. "Passkey" probably gives a more "modern" feel than "clé d'accès".

Hi @nicobao 

We had the same discussion on our side, and I am still uncertain about which option would be easier to understand or more commonly used. While we leaned slightly towards using "passkey," we are open to reconsider if "clé d'accès" is a term that would be more accessible to individuals without a technical background.	

Thank you for your input!

**Comment by nicobao** [comment]
I typed "qu'est ce que l'autorisation par clés d'accès" puis "qu'est ce que l'autorisation par passkey" on Google.
The first search didn't show any relevant links.
The second immediately showed articles in French explaining what passkeys are and how it works.
Besides, French language has a lot of anglicisms and typically "pass" and "key" are very easily understandable within the context of the modern French language.
On the other hand, I know that non-tech people are used to the word "clé digitale" when it comes to 2FA for securing banking operation on their banking app. 

All in all, I would lean towards "passkey" like you did.

**Comment by FlxMgdnz** [comment]
That was a great idea, thanks.

image

**Comment by bjoern-m** [comment]
Well, apple... then I guess we have no other choice. :) All agree with "clé d´identification"?

File: frontend/elements/src/i18n/fr.ts [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import { Translation } from "./translations";
+
+export const fr: Translation = {
+  headlines: {
+    error: "Une erreur s'est produite",
+    loginEmail: "Se connecter ou s'inscrire",
+    loginFinished: "Connexion réussie",
+    loginPasscode: "Entrez le code d'accès",
+    loginPassword: "Entrez le mot de passe",
+    registerAuthenticator: "Enregistrer une clé d´identification",
```
Comment (nicobao) [inline frontend/elements/src/i18n/fr.ts:10]: Use `'` instead of `´`: `clé d'identification`

File: frontend/elements/src/i18n/fr.ts [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    passcodeResendAfter: "Demander un nouveau code dans {passcodeResendAfter}",
+    unverifiedEmail: "non vérifiée",
+    primaryEmail: "principale",
+    setAsPrimaryEmail: "Définir comme principale",
+    verify: "Vérifier",
+    delete: "Supprimer",
+    newEmailAddress: "Nouvelle adresse e-mail",
+    newPassword: "Nouveau mot de passe",
+    rename: "Renommer",
+    newPasskeyName: "Nouveau nom de clé d´identification",
```
Comment (nicobao) [inline frontend/elements/src/i18n/fr.ts:89]: same here

**Comment by nicobao** [comment]
> Thanks a lot @nicobao, all occurrences have been corrected, see 5e8c079184d800a5d87d47dc0ef97259aa33e6fb.

Nice! I can't review again but I would approve.

**Review by like-a-bause — APPROVED** [review]

**Pull request merged**